### ISSUE TITLE
RSS customization does not work

### ISSUE THREAD

**Issue by Nowosad** [rstudio/distill#376]
The https://rstudio.github.io/distill/blog.html#rss-feed post explains different ways to customize a distill website, including how to add a full-content RSS feed or create a separate feed for different categories. I have followed these guidelines, but without any success - the output only has one RSS feed with just a summary of the content. I created a minimal website example at 
https://nowosad.github.io/testsite/, source code at https://github.com/Nowosad/testsite.

I also checked many existing distill websites listed at https://distillery.rbind.io/showcase.html, and I was unable to find any with a full content RSS feed or create a separate feed for different categories.

I am using the most recent GitHub version of `distill`.

Related issues:
1. https://github.com/rstudio/distill/issues/99
2. https://github.com/rstudio/distill/issues/33

**Comment by Cdk29** [comment]
Same problem here. Cannot got an adequate RSS flux.

@maelle sorry for the wild tagging but you seem to be the one with the most experience with this issue, is there something missing in the official documentation ?

**Comment by maelle** [comment]
Sorry, I had the most questions, but not the most answers, and I don't use distill regularly. :slightly_smiling_face:

**Comment by Cdk29** [comment]
Well, digging into the recent commits, I found this :

https://github.com/rstudio/distill/commit/6dfc29d4bd771d3fe65be19a2de0ebc396a15fe3 

Which seems to interrupt full rending of article inside the RSS on purpose.

**Comment by Cdk29** [comment]
Well, I have tried to install an old version of distill but did not had any success with it.

Guess I have to put my blog to trash and start a new one.

**Comment by cderv** [comment]
I confirm this is a bug introduced by a change in **rmarkdown**. It took me some time to find but I believe I found it. However, it seems it may not be the only issue. Hopefully, I'll be able to push a fix soon.

**Comment by Cdk29** [comment]
@cderv glad to read it. Is there a temporary work around ?

**Comment by cderv** [comment]
Can you try with the PR branch ? 

````r
remotes::install_github("rstudio/distill#380")
````

and tell me if this fix the issue for you ? 

You may need to re render your post in the blog section.

**Comment by Cdk29** [comment]
Yesssss, it seems to work after rerendering the rmd post. The rss flux seems quite bigger.

I will see if it works in real life conditions after rendering the post with the correct parameters in few hours, but for the mock version on my computer it seems fine.

Merci !

**Comment by cderv** [comment]
Great! 

I let you try that in real life. Do you use categorized feed ? This should work too, and it would be great to have confirmation. 

I'll wait for that before merging. You can confirm me in the PR directly. Thanks!

**Comment by cderv** [comment]
@Nowosad if you happens to pass by here and confirm it work for you too that would be awesome. Thanks!

**Comment by Cdk29** [comment]
Hi.
Sorry to reopen this issue but there is half a problem.

Indeed, when compiling the rmd document, the feed is as expected.

However, if running "build and restart" in RSudio, or, in code :+1: 
`==> rmarkdown::render_site(encoding = 'UTF-8')
`
The RSS feed revert to "normal".

Not sure how it is expected.

**Comment by Nowosad** [comment]
Hi @cderv - thank you for looking at this issue. 
I tested your solution using a simple webpage at https://github.com/Nowosad/testsite. Results:

1. Rerendering an Rmd blog post adds complete content. However...
2. ...the complete content disappears after building the website.
3. Separate feed for different categories still does not work.

**Comment by cderv** [comment]
Thanks for getting back to me. So 2 is the same as #381 then, and 3 may be related. 
I'll look into that.

**Comment by GenChangHSU** [comment]
Hello, 
I have an additional question about the full RSS feed: 
As @Nowosad mentioned, re-rendering individual rmd blog posts (but not building the website again) will generate full contents in the index.xml file. However, when I tested the file in the RSS previewer, the images could not be displayed (broken images). I inspected the web page source and found weird image paths (e.g., something like <img src="file251857b04b6a_files/figure-html/unnamed-chunk-1-1.png" width="70%" />) in the xml file. Wondering why this happened and also how to fix it. Could anyone help me out?

My website: [https://genchanghsu.github.io/ggGallery/](https://genchanghsu.github.io/ggGallery/)
The RSS feeds: [https://genchanghsu.github.io/ggGallery/index.xml](https://genchanghsu.github.io/ggGallery/index.xml)

I am submitting my blog to R-bloggers but now getting stuck with the full RSS feed problem. Thanks so much!
